code-challenges.js

```diff
@@ -58,4 +131,31 @@ const number2 = 100
 
 // Set two:
 const number3 = 27
-const number4 = 24
+const number4 = 24 
+
+//----- I CREATED A FUNCTION THAT TAKE IN A SET setOne AND setTwo OF ANY TWO numbers....
+
+let fourTwo = number2 - number1
+
+let three = number3 - number4
+
+//----- setOne TAKES number1 AND number2... return number2 AND number1 AND SUBTRACTS THE SMALLER number FROM THE LARGER number....
+
+const setOne = (number1, number2) => {
+return number2 - number1
+}
+
+//----- setTwo TAKES number3 AND number4... return number3 AND number4 AND SUBTRACTS THE SMALLER number FROM THE LARGER number....
+
+const setTwo = (number3, number4) => {
+    return number3 - number4
+}
+
+console.log(fourTwo)
+
+console.log(three)
```

Comment:
Could this be turned into a single function that takes in the numbers dynamically?